jobs/rootfs-builder.jpl

```diff
@@ -58,7 +73,14 @@ mv config/rootfs/debos/${config}/* ${pipeline_version}
     }
 }
 
-def upload(config, pipeline_version, kci_core) {
+def upload(config, pipeline_version, kci_core, rootfs_type, arch) {
+    rootfs_upload_path = "images/rootfs/debian/${config}/${pipeline_version}"
+    print("\n Uploading rootfs_type: ${rootfs_type}")
+
+    if (rootfs_type == "buildroot") {
+       rootfs_upload_path = "images/rootfs/buildroot/${config}/${pipeline_version}/${arch}"
```

Comment:
Well, now we have two sub-directories with the architecture:
https://storage.staging.kernelci.org/images/rootfs/buildroot/buildroot-baseline/20211105.2/arm64/arm64/